### PR TITLE
Update Documentation markdown format

### PR DESCRIPTION
**Pull request by natct10**
- [x] README.md 
- [x] Documentation.md
- [x] Add link to doc in the readme

### CODE REVIEW

**Comment by natct10** [comment]
> # [Codecov](https://codecov.io/gh/SubstraFoundation/distributed-learning-contributivity/pull/294?src=pr&el=h1) Report
> 
> > Merging [#294](https://codecov.io/gh/SubstraFoundation/distributed-learning-contributivity/pull/294?src=pr&el=desc) ([2272eff](https://github.com/SubstraFoundation/distributed-learning-contributivity/commit/2272eff3c1a34018e9d520ad5a7bb1dcf7c85049)) into [master](https://codecov.io/gh/SubstraFoundation/distributed-learning-contributivity/commit/b15cf880f4a68deea3aeea4071eaca18f43d9145?el=desc) ([b15cf88](https://github.com/SubstraFoundation/distributed-learning-contributivity/commit/b15cf880f4a68deea3aeea4071eaca18f43d9145)) will **not change** coverage.
> > The diff coverage is `n/a`.
> 
> [Impacted file tree graph](https://codecov.io/gh/SubstraFoundation/distributed-learning-contributivity/pull/294?src=pr&el=tree)
> 
> ```diff
> @@           Coverage Diff           @@
> ##           master     #294   +/-   ##
> =======================================
>   Coverage   43.61%   43.61%           
> =======================================
>   Files          11       11           
>   Lines        2380     2380           
> =======================================
>   Hits         1038     1038           
>   Misses       1342     1342           
> ```
> 
> [Continue to review full report at Codecov](https://codecov.io/gh/SubstraFoundation/distributed-learning-contributivity/pull/294?src=pr&el=continue).
> 
> > **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> > `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> > Powered by [Codecov](https://codecov.io/gh/SubstraFoundation/distributed-learning-contributivity/pull/294?src=pr&el=footer). Last update [b15cf88...2272eff](https://codecov.io/gh/SubstraFoundation/distributed-learning-contributivity/pull/294?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

Neat! cc @RomainGoussault @arthurPignet @bowni

File: mplc/doc/documentation.md [hunk 1]
```diff
@@ -6,57 +6,57 @@ Table of content:
...
+  - [Definition of collaborative scenarios](#definition-of-collaborative-scenarios)
+  - [Configuration of the collaborative and distributed learning](#configuration-of-the-collaborative-and-distributed-learning)
+  - [Configuration of contributivity measurement methods to be tested](#configuration-of-contributivity-measurement-methods-to-be-tested)
+  - [Miscellaneous](#miscellaneous)
 - [Dataset generation](#dataset-generation)
-  * [Dataset](#dataset)
-  * [Model generator](#model-generator)
-  * [Preprocessing](#data-labels)
-  * [Split in train, validation and test sets](#split-in-train-validation-and-test-sets)
+  - [Dataset](#dataset)
```
Comment (arthurPignet) [inline mplc/doc/documentation.md:22]: miss the experiment section ?

File: mplc/doc/documentation.md [hunk 2]
```diff
@@ -532,14 +541,14 @@ The data labels can take whatever shape you need, with only one condition. The l
 
 The `Dataset` object must provide separate train and test sets (referred to as global train set and global test set).
 
-The global train set is then further split into a global train set and a global validation set, by the function `train_val_split_global()`. Please note that if this function is not overwritten, the SKLearn's `train_test_split()` function will be called by default, and `10%` of the training set will be use as validation set. 
+The global train set is then further split into a global train set and a global validation set, by the function `train_val_split_global()`. Please note that if this function is not overwritten, the SKLearn's `train_test_split()` function will be called by default, and `10%` of the training set will be use as validation set.
```
Comment (arthurPignet) [inline mplc/doc/documentation.md:544]: outdated ... Now the test set is fixed by the dataset, and the validation set size can be set via val_proportion
the methods do not exist anymore

File: mplc/doc/documentation.md (hunk 1 above)
Comment (natct10) [reply mplc/doc/documentation.md:23]: Nice, thx!

**Review by arthurPignet — APPROVED** [review]

**Review by arthurPignet — APPROVED** [review]

File: mplc/doc/documentation.md [hunk 3]
```diff
@@ -536,16 +551,9 @@ The data labels can take whatever shape you need, with only one condition. The l
...
 The `Dataset` object must provide separate train and test sets (referred to as global train set and global test set).
 
-The global train set is then further split into a global train set and a global validation set, by the function `train_val_split_global()`. Please note that if this function is not overwritten, the SKLearn's `train_test_split()` function will be called by default, and `10%` of the training set will be use as validation set. 
-
-In the multi-partner learning computations, the global validation set is used for early stopping and the global test set is used for performance evaluation.
-
-The global train set is then split amongst partners (according to the scenario configuration) to populate the partner's local datasets.
-
-For each partner, the local dataset will be split into separated train, validation and test sets, using the `train_test_split_local()` and `train_val_split_local()` functions.
-These are not mandatory, by default the local dataset will not be split.
```
Comment (RomainGoussault) [inline mplc/doc/documentation.md:546]: Why was this removed ?

File: mplc/doc/documentation.md [hunk 4]
```diff
@@ -536,16 +551,9 @@ The data labels can take whatever shape you need, with only one condition. The l
...
 
-The global train set is then further split into a global train set and a global validation set, by the function `train_val_split_global()`. Please note that if this function is not overwritten, the SKLearn's `train_test_split()` function will be called by default, and `10%` of the training set will be use as validation set. 
-
-In the multi-partner learning computations, the global validation set is used for early stopping and the global test set is used for performance evaluation.
-
-The global train set is then split amongst partners (according to the scenario configuration) to populate the partner's local datasets.
-
-For each partner, the local dataset will be split into separated train, validation and test sets, using the `train_test_split_local()` and `train_val_split_local()` functions.
-These are not mandatory, by default the local dataset will not be split. 
+The global train set is then further split into a global train set and a global validation set, in a stratified way. Please note that you can specified the proportion of the train set used for validation via the `val_proportion` keyword argument. Default is `0.10`.
```
Comment (RomainGoussault) [inline mplc/doc/documentation.md:554]: I would remove `in a stratified way` as we don't use any stratification strategy for the global and train/val set ?

**Review by RomainGoussault — CHANGES_REQUESTED** [review]

File: mplc/doc/documentation.md (hunk 3 above)
Comment (arthurPignet) [reply mplc/doc/documentation.md:546]: Because it's outdated, see 288

File: mplc/doc/documentation.md (hunk 4 above)
Comment (arthurPignet) [reply mplc/doc/documentation.md:554]: We do now, as some scenario have unbalanced label representation

Comment (arthurPignet) [reply mplc/doc/documentation.md:554]: same, see 288

Comment (RomainGoussault) [reply mplc/doc/documentation.md:554]: Ah nice !!!

**Review by RomainGoussault — APPROVED** [review]
